src/index.html

```diff
@@ -4,14 +4,169 @@
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form_fieldset">
+        <legend>Registration</legend>
+
+        <div class="form__input">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="form__input">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="8"
+            maxlength="24"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form_fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__input">
+          Do you love cats?
+          <input
+            type="radio"
+            id="catsChoiceYes"
+            name="catsChoice"
+            value="yes"
+          >
+          <label for="catsChoiceYes">Yes</label>
+          <input
+            type="radio"
+            id="catsChoiceNo"
+            name="catsChoice"
+            value="no"
+          >
+          <label for="catsChoiceNo">No</label>
+        </div>
+        <div class="form__input">
+          <label for="colors">What is your favorite color?</label>
+          <input
+            type="color"
+            name="colors"
+            id="colors"
+          >
+        </div>
+        <div class="form__input">
+          <label for="sleepTime">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="sleepTime"
+            id="sleepTime"
+          >
+        </div>
+        <div class="form__input">
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+        <div class="form__input">
+          <label for="rate">How do you rate our work?</label>
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+            value="0"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form_fieldset">
+        <legend>Additional info:</legend>
+
+        <div class="form__input">
+          <label for="comments">Comments:</label>
+          <textarea name="comments" id="comments"></textarea>
+        </div>
+        <div class="form__input">
+          <label for="recommend">Would you recommend us?</label>
+          <select name="recommend" id="recommend">
+            <option value="choiceYes">yes</option>
+            <option value="choiseNo">no</option>
+          </select>
+        </div>
+      </fieldset>
+
+      <button type="submit" class="form__button">Submit</button>
```

Comment:
Useless class, remove it
```suggestion
      <button type="submit">Submit</button>
```